index.js

```diff
@@ -16,17 +16,16 @@ module.exports = (host, method, port) => {
 		host: host,
 		method: method || 'HEAD',
 		port: port || 443,
-		rejectUnauthorized: false,
-		agent: false,
```

Comment:
This was a fix, please check #7

Reply:
This was not intended... I might have branched from the wrong branch (master)... I can fix it back in... It is just the single agent line isn't it?

WE should add a test case 😜